docs/UserGuide.md

```diff
@@ -253,20 +253,47 @@ Examples:
 
 Sorts the applicant list by using a particular descriptor as the sorting criteria.
 
-Format: `sort d/DESCRIPTOR`
+Format: `sort d/DESCRIPTOR [dsc/]`
 * `DESCRIPTOR` must be either `name` or `phone` or `email` or `position` or `score` or `status`.
+* `dsc/` flag indicates to sort the applicant list in descending order.
 
 Examples:
 * `sort d/name` sorts the applicant list by name in alphabetical order.
 * `sort d/phone` sorts the applicant list by phone numbers in ascending order.
 * `sort d/email` sorts the applicant list by email in alphabetical order.
-* `sort d/position` sorts the applicant list by positions in alphabetical order.
-* `sort d/score` sorts the applicant list by score in descending order.
-* `sort d/status` sorts the applicant list by status in alphabetical order.
+* `sort d/position dsc/` sorts the applicant list by positions in descending alphabetical order.
+* `sort d/score dsc/` sorts the applicant list by score in descending order.
+* `sort d/status` sorts the applicant list by status in the order `UNDECIDED`, `OFFERED`, `REJECTED`.
 
 UI mockup:
 ![Sort UI Mockup](images/user-guide/sort.png)
 
+---
+<a name="filter"></a>
+
+### `filter`: Filtering applicants by fields
+
+Filters the applicant list based on fields given by the user.
+
+Format: `filter [n/Name] [e/Email] [p/Position] [hp/Phone] [s/Status] [lts/Score] [gts/Score]`
+* All fields are optional, however at least one of the optional fields must be provided.
+* Any combination of multiple different fields is allowed. 
+* Only one of each field can be provided.
+* Only applicants matching all fields will be returned. (i.e. AND search).
+* for `[n/Name]` field, only applicants whose name contains the full substring will be returned, e.g. `n/Ivan Chew` will **NOT** return `Ivan Lee`.
+* `[n/Name]` `[e/Email]` `[p/Position]` fields are case-insensitive, e.g. `n/JOHN` will return `john`.
+* `[lts/Score]` `[gts/Score]` fields include equality in filters, e.g. `gts/7` will return all applicants with a greater than or equal to `7`.
```

Comment:
It seems like the current implementation does not include equality 😮 Perhaps possible to add it as a future implementation 😎 but better to describe the current implementation here and exclude the part on equality 😄

Reply:
Seems like a bug, as the implementation was supposed to handle equality 🥸. But will fix that next week! For now I have changed the UG to reflect not equality!